rero_ils/modules/notifications/api.py

```diff
@@ -104,11 +99,13 @@ def create(cls, data, id_=None, delete_pid=False,
             delete_pid=delete_pid)
         return record
 
-    def update_process_date(self):
+    def update_process_date(self, send=True):
         """Update process date."""
-        self['process_date'] = datetime.now(timezone.utc).isoformat()
-        self.update(data=self.dumps(), dbcommit=True, reindex=True)
-        return self
+        if send:
+            self['process_date'] = datetime.utcnow().isoformat()
+        else:
+            self['update_not_send_date'] = datetime.utcnow().isoformat()
```

Comment:
This is not `update_not_send_date` but only `not_send_date`.

Additionally, maybe we could keep `process_date` in both case but add a `send` boolean value into ressource instead of using "not_send_date" (seems not very understandable in a couple of months)

Reply:
changed to `notification_sent` boolean